contrib/mypy/src/python/pants/contrib/mypy/tasks/mypy_task.py

```diff
@@ -60,9 +75,22 @@ def is_non_synthetic_python_target(target):
   def is_python_target(target):
     return isinstance(target, PythonTarget)
 
+  def _is_tagged_target(self, target):
```

Comment:
Type hints would be awesome for this new code.

```suggestion
  def _is_tagged_target(self, target: Target) -> bool:
```